packages/nextjs-template/utils/links.ts

```diff
@@ -4,18 +4,17 @@ import {
   NoteProps,
 } from "@dendronhq/common-all";
 import _ from "lodash";
+import { env } from "../env/client";
 
 export function getNoteUrl(opts: { note: NoteProps; noteIndex: NoteProps }) {
   const { note, noteIndex } = opts;
   return note.id === noteIndex.id ? "/" : `/notes/${note.id}`;
 }
 
 export function getAssetUrl(url: string) {
-  const out =
-    process.env.NODE_ENV === "production" &&
-    process.env.NEXT_PUBLIC_ASSET_PREFIX
-      ? process.env.NEXT_PUBLIC_ASSET_PREFIX + url
-      : url;
```

Comment:
we don't want asset prefix to activate for development (because people are running via `localhost`). so would change this to `!== development` instead

Reply:
[`5e6517f` (#3580)](https://github.com/dendronhq/dendron/pull/3580/commits/5e6517feb819af4d4edd3b1fe0da48cfd40c2525)